.github/workflows/false-positive-approvals.yml

```
@@ -0,0 +1,158 @@
+name: false positive approvals
+
+on:
+  issue_comment:
+    types: [ created ]
+
+jobs:
+  update_suppression:
+    name: Update Suppression Rules
+    if: ${{ !github.event.issue.pull_request && 
+      contains(github.event.issue.labels.*.name, 'FP Report') && 
+      contains(github.event.comment.body,'approved') &&
+      (contains(github.event.comment.user.login, 'jeremylong') ||
+      contains(github.event.comment.user.login, 'aikebah') || 
+      contains(github.event.comment.user.login, 'nhumblot'))  }}
```

```suggestion
      (github.event.comment.user.login === 'jeremylong') ||
      github.event.comment.user.login === 'aikebah') || 
      github.event.comment.user.login === 'nhumblot'))  }}
```

Stronger check required to avoid people with github logins like `aikebah1337` impersonating as me. Derived the alternative check from the filter applied at line 41 to check whether a comment was made by the github actions bot. Not sure whether exact same check can be used for mere mortals with a github account